src/drivers/intel/baytrail/ipc.c

```diff
@@ -172,6 +172,10 @@ int platform_ipc_init(struct ipc *ipc)
 {
 	struct ipc_data *iipc;
 	uint32_t imrd, dir, caps, dev;
+	struct task_ops ops = {
+		.run = ipc_platform_do_cmd,
+		.complete = ipc_platform_complete_cmd,
+	};
```

Comment:
@tlauda ipc_platform_do_cmd and ipc_platform_complete_cmd are already abstracted for each platform. Does it make sense to move this struct definition to the ipc header file?

Reply:
Done.